### PR TITLE
feat(STONEINTG-1006): adding to component status.lastPromotedImage

### PR DESCRIPTION
**Pull request by kasemAlem**
Adding new status field lastPromotedImage , for more details review in [STONEINTG-1006](https://issues.redhat.com/browse/STONEINTG-1006)

### CODE REVIEW

**Comment by ralphbean** [comment]
@Katka92 pinging you here for review and awareness in light of the discussion yesterday about contract testing.

**Review by ralphbean — APPROVED** [review]

File: README.md
Comment (dirgim) [inline README.md:2]: This change probably doesn't need to be part of this PR.

Comment (kasemAlem) [reply README.md]: removed

**Review by dirgim — APPROVED** [review]
LGTM

**Comment by ralphbean** [comment]
@Katka92 any concerns with this before we merge it?

**Review by johnmcollier — APPROVED** [review]

File: README.md
Comment (jsztuka) [inline README.md:2]: Can this be switched back?

**Review by Josh-Everett — APPROVED** [review]
LGTM other than the trailing space nit mentioned by Jiri

File: README.md
Comment (kasemAlem) [reply README.md:2]: It can be , but we (@dirgim ) agreed that it's not part of this PR , right ?

Comment (dirgim) [reply README.md:2]: There's still an added space there, I think @jsztuka is referring to that.

**Comment by kasemAlem** [comment]
> @ralphbean Thank you for pinging me. @rohitkrai03 can you please confirm it is not a breaking change for the UI?

@rohitkrai03 , @Katka92  we still waiting for a confirmation if it's safe to merge this PR , can you please update ?

**Review by rohitkrai03 — APPROVED** [review]
/approve

**Pull request merged**
